### PR TITLE
Feature tm 2

### PR DESCRIPTION
**Pull request by shadabwahidullah**
# In this milestone I have:

- Created genre class and Implemented all its methods
- Created music_album and Implemented all its methods.

### CODE REVIEW

File: model/genere.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  def initialize(name, id = Random.rand(1..999_999))
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genere = self
```
Comment (Wikitelmex) [inline model/genere.rb:12]: well done in this class, however, in the diagram class, it's named "genre" instead "genere", please change the name, in this class and in the name of the file, also in item file.

File: model/item.rb [hunk 2]
```diff
@@ -1,13 +1,9 @@
 class Item
-  attr_reader :id, :archived
+  attr_reader :genere, :author, :source, :label
```
Comment (Wikitelmex) [inline model/item.rb:2]: well done in this class, however, in the diagram class, it's named "genre" instead "genere", please change the name.

**Review by Wikitelmex — CHANGES_REQUESTED** [review]
Hi @shadabwahidullah, 

Excellent work so far! 👌
You have implemented all the required features correctly and all the tests are passing. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [ ] Please, correct some mistypes.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: model/genere.rb (hunk 1 above)
Comment (shadabwahidullah) [reply model/genere.rb]: Thanks for mentioning

File: model/item.rb [hunk 3]
```diff
@@ -18,4 +14,9 @@ def can_be_archived?
   def move_to_archive
     @archived = true if can_be_archived? == true
   end
+
+  def genere=(genere)
+    @genere = genere
```
[the remark this answers is not in the document]
Comment (shadabwahidullah) [reply model/item.rb]: Thanks for mentioning it.

File: model/item.rb (hunk 2 above)
Comment (shadabwahidullah) [reply model/item.rb]: Thanks for mentioning it.

**Pull request merged**